src/psyclone/psyir/nodes/loop.py

```diff
@@ -43,6 +44,10 @@
 from psyclone.psyir.nodes.statement import Statement
 from psyclone.psyir.nodes.routine import Routine
 from psyclone.psyir.nodes import Schedule
+from psyclone.psyir.nodes.assignment import Assignment
+from psyclone.psyir.nodes.intrinsic_call import IntrinsicCall
+from psyclone.psyir.nodes.literal import Literal
+from psyclone.psyir.nodes.reference import Reference
 from psyclone.psyir.symbols import ScalarType, DataSymbol
 from psyclone.core import AccessType, Signature
```

Comment:
Not yours but would you mind putting all of these in alpha order in fact?

Reply:
Addressed in f703b5df0.

Reply:
Thanks.